include/bout/coordinates.hxx

```diff
@@ -35,10 +35,13 @@ class Coordinates;
 #ifndef __COORDINATES_H__
 #define __COORDINATES_H__
 
-#include "mesh.hxx"
-#include "datafile.hxx"
-#include "utils.hxx"
 #include <bout_types.hxx>
+#include <field2d.hxx>
+#include <field3d.hxx>
+#include <fieldperp.hxx>
```

Comment:
I'm confused about this... should we use `<>` for system headers and `""` for BOUT++ headers?

Reply:
I tend to view `<>` as being something provided by a library (including `bout`) and `""` for local files. Or in other words `<>` for things in an include path and `""` for things that aren't. This is just my interpretation though. I think `""` _can_ search more places, but if this is an advantage or a disadvantage may be a matter of opinion.

Reply:
One reason to prefer `""` for all BOUT++ headers: clang-format and include-what-you-use can group headers together, and it might be nice to separate the system headers from ours.